.github/workflows/windows-tests.yaml

```diff
@@ -43,12 +45,12 @@ jobs:
           cache: "pip"
           cache-dependency-path: "requirements*.txt"
 
-      - name: Install packages
-        run: |
-          python -m pip install --upgrade pip
-          pip install --upgrade --upgrade-strategy eager -e .[dev]
+            - name: Install packages
```

Comment:
It looks like this workflow also had a few spacing issues introduced!

Reply:
Should be updated now!